src/components/gltf-model-plus.js

```diff
@@ -407,8 +414,16 @@ AFRAME.registerComponent("gltf-model-plus", {
         }
       });
 
+      this.el.object3D.traverse(obj => {
+        obj.frustumCulled = false;
+      });
       this.el.emit("model-loaded", { format: "gltf", model: this.model });
+      await AFRAME.scenes[0].systems.ticker.nextTick();
+      this.el.object3D.traverse(obj => {
+        obj.frustumCulled = true;
+      });
```

Comment:
Is it correct to emit `model-loaded` before we turn frustumCulling back on? If the aim here is to wait for the model to be uploaded to the GPU, perhaps it's better not to claim `model-loaded` until the GPU upload is complete.